skfda/exploratory/depth_measures.py

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
I would call this module `depth`.

Reply:
I though about that. I do not have a strong opinion either way.

Reply:
Done in 0a3284ccbcda2bdd00249a21a57d2db127ef6703.